Script patch staging (#979)
pass-culture/pass-culture-app-native: doc/deployment/deployment.md

If I modified native code, I need to hard deploy:

- `yarn trigger:testing:deploy:patch`

>   This will create a patch tag `testing_vX.X.X` and push it.

Reviewer: ```suggestion
  This will bump the patch number, create a tag `testing_vX.X.X+1` and push it.
```

pour que ce soit plus explicite

### Hard deploy (once a week, manual)

When you want to deploy the current version of master in staging, you can run the following commands:

- `yarn trigger:staging:deploy`

> This will create a tag minor `vX.X.X` and push it.

Reviewer: ```suggestion
This will bump the `minor` version, create a tag `vX.X+1.X` and push it.
```

- or `trigger:staging:deploy:patch`

>   This will create a patch minor `vX.X.X` and push it.

Reviewer: ```suggestion
  This will bump the `patch` version, create a  tag `vX.X.X+1` and push it.
```
